Utilities/DataLoading/EdFi.LoadTools.Test/MapXmlLookupToGetByExampleJsonStepTests.cs

```diff
@@ -9,16 +9,16 @@
 using EdFi.LoadTools.Engine.Mapping;
 using EdFi.LoadTools.Engine.XmlLookupPipeline;
 using EdFi.LoadTools.Test.MappingFactories;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NUnit.Framework;
 
 namespace EdFi.LoadTools.Test
 {
-    [TestClass]
+    [TestFixture]
     public class MapXmlLookupToGetByExampleJsonStepTests
     {
         private MapXmlLookupToGetByExampleJsonStep _mapStep;
 
-        [TestInitialize]
+        [SetUp]
```

Comment:
i think you may have the wrong attribute here. https://github.com/nunit/docs/wiki/SetUp-and-TearDown

Reply:
Setup method is correct for TextFixture class  . Please refer https://github.com/nunit/docs/wiki/SetUp-Attribute

Reply:
I changed to OneTimeSetup to run only once per test run . I agree with you